common/model-views.cpp

```diff
@@ -1911,30 +1926,51 @@ namespace rs2
         return results;
     }
 
-    bool is_resolution_max(const std::vector<std::pair<int, int>>& res_values, int width, int height, const std::pair<int, int>& calibration_res)
+    bool subdevice_model::is_d405_oem_cal_profile(int d405_calibration_width, int d405_calibration_height) const
```

Comment:
SKU name can be dropped- applies to all SKUs

Reply:
Done